src/Auth.ts

```diff
@@ -636,8 +636,8 @@ export class Auth {
       resource = resource.substr(0, pos);
     }
 
-    if (resource === 'https://api.bap.microsoft.com') {
-      // api.bap.microsoft.com is not a valid resource
+    if (resource === 'https://api.bap.microsoft.com' || resource === 'https://api.powerapps.com') {
```

Comment:
Do we use `api.bap.microsoft.com` anywhere else? If not, let's remove it altogether.

Reply:
Yes, it's being used in some Power Platform commands, like the Flow export.